regression/scripts/aerodynamics/aerodynamics.py

```diff
@@ -175,10 +229,10 @@ def main():
     drag_tests.cd_p_wing      = np.abs((cd_p_wing - cd_p_wing_r)/cd_p_wing)
     drag_tests.cd_tot         = np.abs((cd_tot - cd_tot_r)/cd_tot)
     
-    print '\nCompute Drag Test Results\n'    
-    print 'cd_tot=', cd_tot
+    print('\nCompute Drag Test Results\n')    
+    print('cd_tot=', cd_tot)
    
-    for i, tests in drag_tests.items(): 
+    for i, tests in list(drag_tests.items()): 
```

Comment:
do we lose code efficiency here with the extra function call?  if so, can we circumvent?

Reply:
Good point. This was an automatic update from lib2to3. I don't think it's necessary for .items()